openfe/tests/protocols/test_rfe_tokenization.py

```diff
@@ -36,15 +44,30 @@ class TestRelativeHybridTopologyProtocolResult(GufeTokenizableTestsMixin):
     def instance(self):
         pass
 
+class TestRelativeHybridTopologyProtocolOtherUnits(GufeTokenizableTestsMixin):
```

Comment:
I was looking at the tests earlier today and stumbled upon this - the name is really confusing. Can we maybe consider switching it out to something else? (especially having it in tokenization just threw me off).